lib/broccoli/glimmer-app.ts

```diff
@@ -364,6 +381,23 @@ Please run the following to resolve this warning:
     return this.package(jsTree, cssTree, publicTree, htmlTree);
   }
 
+  private lint() {
+    let { src } = this.trees;
+
+    let templateTree = new Funnel(src, {
+      include: ['**/*.hbs']
+    });
+
+    let srcTree = new Funnel(src, {
+      exclude: ['**/*.hbs']
+    });
+
+    let lintedTemplates = addonLintTree(this.project, 'templates', templateTree);
```

Comment:
I _vaguely_ recall thus being singular in Ember CLI’s pipeline, can you double check?

Reply:
Just rechecked: It is also `templates` in Ember CLI (https://github.com/ember-cli/ember-cli/blob/35536c7c378751a406766e836081aa47971d250b/lib/broccoli/ember-app.js#L1346)

Reply:
Awesome, thank you (and sorry for the run around)...